src/jsc/jsc_value.hpp

```diff
@@ -177,8 +177,7 @@ inline double jsc::Value::to_number(JSContextRef ctx, const JSValueRef &value) {
         throw jsc::Exception(ctx, exception);
     }
     if (isnan(number)) {
-        throw std::invalid_argument(util::format("Value '%1' not convertible to a number.",
-                                                 (std::string)to_string(ctx, value)));
+        std::nan("");
```

Comment:
```suggestion
        return std::nan("");
```

Reply:
Seems like we should add some tests that would have detected this?

Reply:
I think the RN tests will detect it. Locally, I only ran the test using node.js.

Reply:
I mean, `std::nan("")` doesn't do anything... You at least need to return the value.
But as TG writes, why not remove it altogether - unless you can show with a test that it actually does something?